### PR TITLE
Review comments from test run

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,364 @@
...
+
+    @PostConstruct
+    public void joinFirstRound() {
+        System.out.println("Hello world!");
+    }
+}
+```
+When you save this code change, the app should restart again and you should see the "Hello world!" message in your messages.log
+
+## IF you get stuck
```
Comment (realModusOperandi) [inline README.md]: Caps?

File: README.md [hunk 2]
```diff
@@ -1 +1,364 @@
...
+```java
+// don't forget to add these 2 annotations!
+@Inject
+@RestClient
+GameServiceClient gameService;
+
+public void joinRound() {
+    System.out.println("Attempting to register with game service...");
+
+    // TODO: use MPRestClient to pull the valid partyID from the game service
```
Comment (olendvcook) [inline README.md]: the TODO should probably be taken out since you are implementing it in the provided code block.

File: README.md [hunk 3]
```diff
@@ -1 +1,364 @@
...
+- At each game tick "look" in each direction -- forward, left, and right. Compute which direction you can go in a straight line for the longest, and go that way.
+
+# 5) Let's Get Ready to Rumble!!!
+
+Towards the end of the lab we will host a competition where everyone can battle their AI bots against each other to see who has the best one! In order to do this, you will run just your AI bot on your laptop. You may now shutdown the core services running on your laptop by running `./gradlew stop` from the root of the liberty-bikes repo.
+
+The lab admins should be running a set of core services on their laptops, and will provde you with their IP address. We will call it `$ADMIN_IP` in these instructions.
+
+In order to configure your bot to connect to the lab admins set of core services, you must do the following:
+1. Go to their frontend at `http://$ADMIN_IP:8080 and register your bot and take note of the generated key like you did at the end of section 2
```
Comment (olendvcook) [inline README.md]: Go to their frontend at `http://$ADMIN_IP:8080 -> **port should be 12000**

File: README.md (hunk 1 above)
Comment (aguibert) [reply README.md]: yea, that's intentional. Some people were going through these steps as part of the regular procedure even though they weren't stuck

File: README.md (hunk 3 above)
Comment (aguibert) [reply README.md]: oops, forgot to save the file after editing locally

**Review by olendvcook — APPROVED** [review]

**Pull request merged**